packages/reputation-miner/ReputationMiner.js

```diff
@@ -734,7 +734,7 @@ class ReputationMiner {
 
   async getEntryIndex(startIndex = 1) {
     // Get how much we've staked, and thefore how many entries we have
-    const [stakeAmount] = await this.colonyNetwork.getMiningStake(this.minerAddress);
+    const stakeAmount = await this.getMiningStake();
```

Comment:
How does this work? `getMiningStake` does not return a single value?

Reply:
Mm good point, should be `minerStake.amount`